components/Buttons/examples/ButtonsContentEdgeInsetsExample.m

```diff
@@ -56,17 +57,6 @@ - (void)viewDidLoad {
   [self updateInkStyle:self.inkBoundingSwitch.isOn ? MDCInkStyleBounded : MDCInkStyleUnbounded];
 }
 
-- (MDCContainerScheme *)containerScheme {
```

Comment:
Don't you need to load the defaults in `viewDidLoad` (or similar) so it's non-nil in Dragons?

Reply:
I believe we are going to add the containerScheme _injection_ to Dragons. In the mean time, I think Dragons being themed wrong is okay.

Reply:
Since our applyTheme methods don't allow `nil` parameters, at some point you'll have to make sure the property is non-nil even in the Dragon.

Reply:
Agreed at some point in the near future we will need a similar PR to #6458 but I don't think this is blocked by that, would you agree?

Reply:
Adding it as a property set in the initializer addresses my concerns. Thanks!